internal/db/code.go

```diff
@@ -40,14 +43,17 @@ var codeMap map[uuid.UUID]Code = map[uuid.UUID]Code{
 	URLValidator.ID:          URLValidator,
 }
 
-func CallValidator(c Code, arg interface{}) (interface{}, error) {
+func (c Code) Invoke(arg interface{}) (interface{}, error) {
```

Comment:
I'm still not sure I love this switch statement (1) existing and (2) being inside of the DB.

I think an alternative pattern might be:

`aft/internal/starlark/` provides the "starlark runtime" to the server
That could mean it provides a `CodeToFunctionHandle` method keyed by the runtimeID (ie your Runtime enum) to a service (ie struct created at server bringup) called the 'executor' 

Invoke then just becomes executor.invoke(Code)
executor just knows how to lookup a CodeToFunctionHandle, call it, and then invoke the handle.

wdyt?

Reply:
Took a little thinking, but done!